README.md

```diff
@@ -78,6 +78,7 @@ installed first.
 - A supported SYCL implementation, either
     - [hipSYCL](https://github.com/illuhad/hipsycl),
     - [DPC++](https://github.com/intel/llvm)
```

Comment:
```suggestion
    - [DPC++](https://github.com/intel/llvm), or
```

This is a good opportunity to fix something that bugged me about the readme ;)

Reply:
Done.